recipe/meta.yaml

```diff
@@ -23,24 +23,31 @@ requirements:
   run:
     - python
     - async_generator >=1.9
```

Comment:
`async_generator >=1.9` seems needed only for tests 

https://github.com/search?q=repo%3Apython-trio%2Ftrio%20async_generator&type=code

https://github.com/python-trio/trio/blob/b9851229f095ea4aa38c470b425c1a5341fe512f/pyproject.toml#L38-L51

Reply:
Thank you! Addressed.